### PR TITLE
feat: new list styles

### PR DESCRIPTION
**Pull request by jrbromberg**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #115

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [ ] My update follows the CONTRIBUTING guidelines of this project
- [ ] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: src/utilityClasses/_listProperties.scss [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+#{$scope}.#{$prefix}util_listAuroIcon {
+  padding-left: 2*$auro-size-200;
+  list-style: none;
+  li {
+    text-indent: -$auro-size-200;
+  }
+  auro-icon{
+    width: $auro-size-200;
+  }
+}
```
Comment (blackfalcon) [inline src/utilityClasses/_listProperties.scss:191]: Please add a new line at the end of the file.

File: src/utilityClasses/_listProperties.scss [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+  margin-left: 1px;
+  li {
+    text-indent: -$auro-size-200;
+  }
+  li::before {
+    content: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='6' height='6'><circle cx='3' cy='2' r='2'></circle></svg>");
+    display: inline-block;
+    width: $auro-size-200;
+  }
+}
```
Comment (blackfalcon) [inline src/utilityClasses/_listProperties.scss:62]: @leeejune do we have a design spec that this is developed against?

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: src/utilityClasses/_listProperties.scss (hunk 2 above)
Comment (leeejune) [reply src/utilityClasses/_listProperties.scss:62]: @blackfalcon I am not aware of any specs!

**Comment by braven112** [comment]
Closing in favor of this PR with the update design https://github.com/AlaskaAirlines/WebCoreStyleSheets/pull/138

**Pull request closed without merging**